R/import_methods.R

```diff
@@ -218,24 +218,17 @@ function(file,
 #' @importFrom haven read_dta
 #' @export
 .import.rio_dta <- function(file, haven = TRUE,
-                            convert.dates = TRUE,
-                            convert.factors = FALSE,
-                            missing.type = FALSE, ...) {
-    if (isTRUE(haven)) {
-        a <- list(...)
-        if (length(a)) {
-            warning("File imported using haven. Arguments to '...' ignored.")
-        }
-        standardize_attributes(haven::read_dta(file = file))
-    } else {
-        out <- foreign::read.dta(file = file,
-                    convert.dates = convert.dates,
-                    convert.factors = convert.factors,
-                    missing.type = missing.type, ...)
-        attr(out, "expansion.fields") <- NULL
-        attr(out, "time.stamp") <- NULL
-        standardize_attributes(out)
-    }
+                               convert.factors = FALSE,...) {
```

Comment:
The reason I removed all the other `read.dta` specific arguments is that the rest of them matched what were already the default values for `foreign::read.dta`.